ansibullbot/triagers/ansible.py

```diff
@@ -1285,6 +1286,18 @@ def create_actions(self, iw, actions):
                 if label_name in iw.labels:
                     actions.unlabel.append(label_name)
 
+        # docs_only?
+        # https://github.com/ansible/ansibullbot/issues/1047
+        if iw.is_pullrequest():
+            label_name = 'docs_only'
+            if self.meta['is_docs_only']:
+                if label_name not in iw.labels:
+                    actions.newlabel.append(label_name)
+            else:
+                human_labeled = iw.history.was_labeled(label_name, bots=self.BOTNAMES)
+                if label_name in iw.labels and not human_labeled:
```

Comment:
With this we could end up with the following undesired situation:
1. a human adds `docs_only` label
2. a submitter adds a new commit that changes code -> `is_docs_only` is set to `False`
3. ansibot does not remove `docs_only` label because a human added `docs_only` label in the past.

If we want to include the "human_labaled" check we need to be more careful about it:
* was it unlabeled by human afterwards?
* was there a new commit after human added the label?

It might be just easier to not take `human_labeled` into account but I am not against implementing it.

Reply:
Doh! I forgot about this one. I can take it out and just follow the norm of removing as the others do.